config/workflows/gisAssemblyWF.xml

```diff
@@ -43,9 +43,9 @@
     <prereq>normalize-data</prereq>
     <label>Extract bounding box from data for MODS record</label>
   </process>
-  <process name="generate-content-metadata">
+  <process name="generate-structural">
     <prereq>extract-boundingbox</prereq>
-    <label>Generate contentMetadata manifest</label>
+    <label>Generate cocina structural</label>
   </process>
   <process name="finish-gis-assembly-workflow">
     <prereq>generate-content-metadata</prereq>
```

Comment:
I think this should be `generate-structural`?

Reply:
Good catch, fixed.